relay-dynamic-config/src/metrics.rs

```diff
@@ -79,7 +79,7 @@ pub struct CustomMeasurementConfig {
 /// Maximum supported version of metrics extraction from transactions.
 ///
 /// The version is an integer scalar, incremented by one on each new version.
-const TRANSACTION_EXTRACT_VERSION: u16 = 2;
+const TRANSACTION_EXTRACT_VERSION: u16 = 3;
```

Comment:
Please sync with @jan-auer on this version bump.

Reply:
Note that I'm also listing changes to the versions in the doc comment now. Your PR will likely merge first, so I'll take care of listing this change, too.